scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -14,19 +14,23 @@
 # docker run -it  -v <insert sof dir here>:/home/sof/work/sof.git --user `id -u` sof ./incremental.sh
 #
 
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 ARG UID=1000
 
-# Set up proxy from host
-COPY apt.conf /etc/apt/
 ARG host_http_proxy
 ARG host_https_proxy
 ENV http_proxy $host_http_proxy
 ENV https_proxy $host_https_proxy
 
+# for non-interactive package install
+ARG DEBIAN_FRONTEND=noninteractive
+
 RUN apt-get -y update && \
 	    apt-get install -y \
-	    	autoconf \
+		dialog \
+		apt-utils \
+		vim \
```

Comment:
Can I have emacs too? Just kidding, however `jed` would be nice.

Reply:
jed is added